app/src/main/java/openfoodfacts/github/scrachx/openfood/features/LoginActivity.kt

```diff
@@ -158,6 +160,12 @@ class LoginActivity : BaseActivity() {
         _binding = ActivityLoginBinding.inflate(layoutInflater)
         setContentView(binding.root)
 
+        // check flavour and show helper text for user account
+        if (!AppFlavors.isFlavors(AppFlavors.OFF)) {
```

Comment:
Could you import `isFlavors` and `AppFlavors.OFF`?

Reply:
Can you elaborate. I didn't get you.

Reply:
Could you add as a static import the method `isFlavors`, and do the same with `AppFlavors.OFF`?

Reply:
Done.